src/main/java/listeners/MainEventListener.java

```diff
@@ -97,6 +98,7 @@ public void onGuildMessageReceived(@NotNull GuildMessageReceivedEvent event) {
 
         Member member = Objects.requireNonNull(event.getMember());
         TextChannel channel = event.getChannel();
+        Member[] mentions = event.getMessage().getMentionedMembers().toArray(new Member[0]);
```

Comment:
What is the purpose of this? Is this just so we can access the mentee in `kick`?

Reply:
Yep. Right now, if I have a command `$kick @cheesegreater#0001`, `args[0] == "@cheesegreater`. It's really hard to convert that string to a Member, especially if people have identical usernames, so this allows us to access the Member objects in `kick()`.